src/Errors/message.js

```diff
@@ -0,0 +1,44 @@
+export const ErrorMessage = {
+  masterKeyReadOnly: () => 'Cannot perform a write operation when using readOnlyMasterKey.',
+  noEmpty: (field, isField = true) =>
+    `${isField === true ? 'The field ' : ''}${field} must not be empty, null or undefined.`,
+  invalid: (field, isField = true) => `${isField === true ? 'The field' : ''} ${field} is invalid.`,
+  unauthorizedAccess: (type, value) => `This user is not allowed to access ${type}: ${value}`,
+  notFound: type => `${type} not found.`,
+  exists: (type, key) => `${type} ${key} already exists`,
+  required: (field, op = ' to save/update the object', isField = false) =>
+    `A value for ${isField === true ? 'field ' : ''}${field} is required${op}.`,
```

Comment:
This looks a bit difficult to understand, can give examples for the different error messages that this should be used for?

Reply:
op has a default value which has appended space to it. This message is can have an optional op appended to it. I have reverted it to check if op is provided and print a blank space before op.